app/controllers/v0/sessions_controller.rb

```diff
@@ -126,7 +127,7 @@ def persist_session_and_user
 
     def handle_login_error
       fail_handler = SAML::AuthFailHandler.new(@saml_response, @current_user, @session)
-      StatsD.increment("api.auth.login_callback.#{context_key}.failure")
+      StatsD.increment(STATSD_CALLBACK_KEY, tags: ['status:failure', "context:#{context_key}"])
```

Comment:
eventually we might want to track an enum for `fail_handler` known error as well. That being said, `known_errors` should be tracked granularly in sentry already.